code/game/mecha/equipment/tools/other_tools.dm

```diff
@@ -9,14 +9,15 @@
 	icon_state = "mecha_teleport"
 	origin_tech = "bluespace=10"
 	equip_cooldown = 150
-	energy_drain = 1000
+	energy_drain = 8000
```

Comment:
How about a subtype that comes without power drain, for the admin-only machines?